packages/headless/src/controllers/commerce/field-suggestions/headless-category-field-suggestions.ts

```diff
@@ -82,12 +94,23 @@ export function buildCategoryFieldSuggestions(
   });
 
   const getFacetForFieldSuggestions = (facetId: string) => {
-    return engine[stateKey].fieldSuggestionsOrder!.find(
+    return engine[stateKey].fieldSuggestionsOrder.find(
```

Comment:
Not a big deal, but since you're doing `engine[stateKey]` twice in `buildCategoryFieldSuggestions`, maybe you could instead use get state function:

```typescript
const getState = () => engine[stateKey]

// ...
  const getFacetForFieldSuggestions = (facetId: string) => {
    return getState().fieldSuggestionsOrder.find(
      // ...
    )
  }
``` 

Reply:
The same comment applies to the field suggestions controller